### ISSUE TITLE
Getting error from Amazon when I accepted the security profile term & condition.

### ISSUE THREAD

**Issue by vw2k3** [willblaschko/AlexaAndroid#6]
Like @TheOrangeTester told in Issue#5, I got the same error from Amazon when I accepted the security profile term & condition.

[The error screen when click 'I agree'](http://imgur.com/a/dIpyr)

It's already 3 days pass, still getting this error from Amazon@@

**Comment by willblaschko** [comment]
I'm going to close this and keep #7 open for further investigation.

**Comment by TheOrangeTester** [comment]
Now I am not getting such error but new updated sample code getting crashed after I click on continue on security permission page.(which opens in webview)

**Comment by willblaschko** [comment]
Did you get the log cat for the crash?

Also, try the sample app in the Play Store (link on the repo page), and see
if that crashes in the same place for you.  Thanks!

**Comment by TheOrangeTester** [comment]
App is not compatible with my device.

**Comment by willblaschko** [comment]
Minimum API level too high?

**Comment by TheOrangeTester** [comment]
In Play store I can see that App can work from 4.0.3 
My  
minSdkVersion 15 & targetSdkVersion 23

**Comment by TheOrangeTester** [comment]
Application is getting crashed at ,

LoginWebViewActivity.java

 WebViewClient mWebViewClient = new WebViewClient(){
        @Override
        public boolean shouldOverrideUrlLoading(WebView view, String url) {
            if(BuildConfig.DEBUG){
                Log.i(TAG, url);
            }
            if(url.startsWith("http") || url.startsWith("https")){
                return super.shouldOverrideUrlLoading(view, url);
            }


//crash is here --->
            Intent i = new Intent();
            i.setData(Uri.parse(url));
            startActivityForResult(i, RESULT_LOGIN);
            
            return false;
        }


    };

**Comment by willblaschko** [comment]
@TheOrangeTester you should be able to install the demo app now.

**Comment by TheOrangeTester** [comment]
@willblaschko  Yes, I was able to download the app. Its wonderful.
But sample code is crashing after app ask for the authentication .
 I have added my key, changed Product ID & application ID.

App is getting crashed when web view receives below url ,
amzn://blazeclan.alexa.app?code=ANXQdTBdDWqWBXBytQVI&scope=alexa%3Aall&state=clientId%3Damzn1.application-oa2-client.535874843ec04b42a7f1b3f1637720f0%26redirectUri%3Damzn%3A%2F%2Fblazeclan.alexa.app%26clientRequestId%3Dd99c5ba9-c26c-4ece-aca8-80c44f0dccfb%26com.amazon.identity.auth.device.authorization.return_auth_code%3Dtrue

At
